index.bs

```diff
@@ -830,13 +831,20 @@ is being used in the <a>text track kind</a> attribute of the <a>text track</a> e
 data kind of a WebVTT file is externally specified, such as in a HTML file's <a>text track</a>
 element. The environment is responsible for interpreting the data correctly.</p>
 
-<p>WebVTT caption or subtitle cues are rendered as overlays on top of a video viewport or into a
-region, which is a subarea of the video viewport.</p>
+<p>WebVTT caption or subtitle cues are rendered as overlays on top of a <a>video viewport</a> or
+into a region, which is a subarea of the video viewport.</p>
+
+
+<h3 id="video-viewport">Video viewport</h3>
+
+<p>The <dfn lt="video viewport">video viewport</dfn> is the video's rendering area which in the case
+of HTML5 is defined by the |video|'s <a>concrete object size</a>.</p>
```

Comment:
The href for the concrete object size link is `https://drafts.csswg.org/css-images-3/#concrete-object-size` but the spec is on /TR at `https://www.w3.org/TR/css3-images/#concrete-object-size` - any reason not to use the latter?

Reply:
That's what bikeshed picked for us. I'll added an explicit ref note so autolinks for preparing the /TR spec will link to the /TR spec.